### ISSUE TITLE
CreateDatabaseDirectory option never works

### ISSUE THREAD

**Issue by vlussenburg** [backtrace-labs/backtrace-unity#24]
Tested on Android, Editor on Mac and iOS.

```
            DatabaseSettings = new BacktraceDatabaseSettings(Configuration);

            Enable = Configuration.Enabled && BacktraceConfiguration.ValidateDatabasePath(Configuration.DatabasePath);
            if (!Enable)
            {
                if (Configuration.Enabled)
                {
                    Debug.LogWarning("Cannot initialize database - invalid path to database. Database is disabled");
                }
                return;
            }
            CreateDatabaseDirectory();
```

If you provide a path that does not exist, the `CreateDatabaseDirectory();` method is never reached.

**Comment by vlussenburg** [comment]
Implemented in the current (sloppy) fork:
https://github.com/vlussenburg/backtrace-unity

**Comment by vlussenburg** [comment]
Tested on:

- [ ] Windows Standalone
- [ ] MacOS Editor
- [ ] MacOs Standalone
- [ ] iOS
- [ ] Android

**Comment by konraddysput** [comment]
This diff should solve this problem: 
https://github.com/backtrace-labs/backtrace-unity/pull/33

**Comment by vlussenburg** [comment]
Tested master @ 9f298881bb070a7da8ac748a0fe701b45becc4c8

- [ ]  Windows Standalone
- [x]  MacOS Editor
- [ ]  MacOs Standalone
- [x]  iOS
- [x]  Android
- [x]  WebGL

**Comment by konraddysput** [comment]
Fix will be available in version 3.0.1 (already on master)
